### DISCUSSION TITLE
Errors when building with XCODE Version 15.0 beta 5 (15A5209g)

### DISCUSSION THREAD

**Discussion by helgehoi** [HaishinKit/HaishinKit.swift#1248] [Q&A]
Hi!

Thank you for providing this kit!

When building with XCODE Version 15.0 beta 5 (15A5209g), there are multiple errors. Most seems to be stemming from the ios target and what is supported features (like ismulticamsupported), but there is also a import UIKit missing in one of the files.

All the best,
Helge

**Comment by shogo4405** [comment]
I couldn't confirm the compilation error on my Xcode 15.0 beta 5. If you are using SPM (Swift Package Manager), I believe this issue might have been addressed. You can find more information about it at: https://github.com/shogo4405/HaishinKit.swift/issues/1246

**Comment by helgehoi** [comment]
Hi! I can confirm that the problems went away now with beta 5 and your UIKit fix. Thank you!
